tools/ci/check_grep2.py

```diff
@@ -151,7 +151,10 @@ def check_href_styles(idx, line):
     if HREF_OLD_STYLE.search(line):
         return [(idx + 1, "BYOND requires internal href links to begin with \"byond://\"")]
 
-EMPTY_LIST_WHITESPACE = re.compile(r"list\(\s+")
+# TODO: This finds most cases except for e.g. `list(1, 2, 3 )`
+# Find a way to include this without breaking macro/tab-aligned versions such as `list(		\`
+# Maybe even make sure it doesn't include comments, idk
```

Comment:
Planning on handling this TODO in the PR?

Reply:
I'm leaving it for better minds than me.